src/test/java/htsjdk/tribble/util/ParsingUtilsTest.java

```diff
@@ -143,6 +164,18 @@ private void tstExists(String path, boolean expectExists) throws IOException{
     }
 
     @Test
```

Comment:
I think we need a way of testing non-file URI's.  I'm not sure the best way to go about it.  One possibility would be to include one of the alternative filesystem providers as a test dependency.  Either that or maybe we could mock one.  There is a [google in memory filesystem] that might make a good test for the non-file based file systems.